lang/python/python3-sqlparse/Makefile

```diff
@@ -0,0 +1,37 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=sqlparse
+PKG_VERSION:=0.3.0
+PKG_RELEASE:=1
+PKG_LICENSE:=BSD
+PKG_MAINTAINER:=Peter Stadler <peter.stadler@student.uibk.ac.at>
+
+PKG_SOURCE:=sqlparse-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/s/sqlparse
+PKG_HASH:=7c3dca29c022744e95b547e867cee89f4fce4373f3549ccd8797d8eb52cdb873
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/sqlparse-$(PKG_VERSION)
+
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-sqlparse
+  SUBMENU:=Python
+  SECTION:=lang
+  CATEGORY:=Languages
+  TITLE:=Non-validating SQL parser module.
+  URL:=https://github.com/andialbrecht/sqlparse
+  DEPENDS:=+python3-light
+  VARIANT:=python3
+endef
+
+define Package/python3-sqlparse/description
+  A non-validating SQL parser module. It provides support for parsing, splitting and formatting SQL statements.
+endef
+
+define Py3Package/python3-sqlparse/install
+# not including sqlformat
```

Comment:
Why?

Reply:
This uses pkg-resources which will pull in about 5MB more. Do you think it would be good to include sqlformat? If so, should I add the dependency on pkg-resources, too?